osu.Server.Queues.ScoreStatisticsProcessor.Tests/DatabaseTest.cs

```diff
@@ -110,7 +110,7 @@ public static ScoreItem CreateTestScore(int? rulesetId = null, int? beatmapId =
                 id = Interlocked.Increment(ref scoreIDSource),
                 user_id = 2,
                 beatmap_id = beatmapId ?? TEST_BEATMAP_ID,
-                ruleset_id = rulesetId ?? 0,
+                ruleset_id = (ushort)(rulesetId ?? 0),

```

Comment:
This is a bit of a random change given that it's in `DatabaseTest` but doesn't hurt I suppose...